cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,93 @@
+/* eslint-disable cypress/unsafe-to-chain-command */
+/* eslint-disable arrow-parens */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+  });
+
+  it('should have pagination', () => {
```

Comment:
add test that will assert clicking on next and previous buttons

Reply:
Done, hope

Reply:
how have you tested "previous" and "next" buttons here?) You just clicked disabled buttons and there was no reaction. You need to add users to increase the total amount of users >=6. Then change amount of users on page to 5. Then click next button and assert that you are on the page 2. Then you need to click previous button and assert that you on the page 1 again.

Reply:
done